modules/local/compare_threshold/main.nf

```diff
@@ -17,18 +17,16 @@ process COMPARE_THRESHOLD {
     script:
     """
     compare_threshold.py \\
-        --sample_id "${meta.id}" \\
+        --sample_id ${meta.id} \\
         --labDataName "${meta.labDataName}" \\
         --libraryType "${meta.libraryType}" \\
         --sequenceSubtype "${meta.sequenceSubtype}" \\
         --genomicStudySubtype "${meta.genomicStudySubtype}" \\
-        --fastp_json "${fastp_json}" \\
-        --mosdepth_global_summary "${summary}" \\
-        --mosdepth_target_regions_bed "${bed}" \\
-        --thresholds "${thresholds}" \\
-        --output "${meta.id}.result.csv"
```

Comment:
Why remove the quotes? Only `--fastp_json` needs the quotes removed because it can be multiple files.

Reply:
those quotes were unnecessary. nexflow can handle them

Reply:
This will break if spaces are ever in sample ID or any of the file names. If we can guarantee this doesn't happen then we don't need the quotes. Otherwise we need the quotes

Reply:
I just edit samplesheet_to_metadata.json accordinly